package/Dockerfile

```diff
@@ -12,4 +12,5 @@ RUN ./scripts/build
 
 FROM registry.suse.com/bci/bci-micro:latest
 COPY --from=builder /usr/src/app/bin/backup-restore-operator  /usr/bin/
+USER 1000:1000
```

Comment:
Could consider using the pattern in:
https://github.com/rancher/prometheus-federator/blob/44a37ffd9d67a4c9f7922eb64e8bdcfe1f4857fd/package/Dockerfile-helm-locker#L10-L15

Just some small "UX" improvements at the OS layer, and then you can change the constructed object's security context  in `templates` to be *slightly* more readable